addons/tts-subtitles/addon.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "Text-to-speech subtitles",
+  "description": "Adds subtitles to every project that uses text-to-speech.",
+  "tags": ["community", "projectPage"],
+  "userscripts": [
+    {
+      "matches": ["projects"],
+      "url": "userscript.js"
+    }
+  ],
+  "userstyles": [
+    {
+      "matches": ["projects"],
+      "url": "userstyle.css"
+    }
+  ],
+  "versionAdded": "1.36.0",
+  "info": [
+    {
+      "id": "add-subtitles",
+      "text": "Only people that have this addon enabled will be able to see these subtitles. In your projects, make sure to add subtitles yourself, so that anyone can use these subtitles.",
```

> make sure to add subtitles yourself

What is this supposed to mean?

---

Probably creating your own subtitles feature for projects you create. Might want to reword that part. It is a good point - it reminds users of this addon who create projects that this isn't necessarily a substitute for making proper built-in subtitles.
```suggestion
      "text": "Only users who have this addon enabled will get subtitles. If you want to offer subtitles to all users in your projects, you should include built-in subtitles.",
```

---

That is what I meant, thanks!